sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -111,6 +112,39 @@ object RapidsPluginUtils extends Logging {
     }
   }
 
+  private def detectMultipleJar(propNames: Seq[String], jarName: String, conf: RapidsConf): Unit = {
+    val classloader = ShimLoader.getShimClassLoader()
+    val rapidsJarURLs = propNames.flatMap(propName =>
+      classloader.getResources(propName).asScala
+    ).toSet
+    lazy val rapidsJars = rapidsJarURLs.map(_.toString.split("!").head).mkString(",")
+    lazy val rapidsJarsVers = rapidsJarURLs.map { 
+      url => scala.io.Source.fromInputStream(url.openStream()).mkString("") 
+    }.mkString(",")
+    lazy val msg = s"Multiple $jarName jars found in the classpath: $rapidsJars, please make " + 
+        s"sure there is only one $jarName jar in the classpath. If it is impossible to fix the " +
+        s"classpath you can suppress the error by setting ${RapidsConf.ALLOW_MULTIPLE_JARS.key}" + 
+        s"to true. Version info: \n$rapidsJarsVers"
+    conf.allowMultipleJars match {
```

Comment:
NIT: boolean case match can be replaced with `if .. else`.
```
if (conf.allowMultipleJars) {
  logWarning(...)
} else {
  ...
}
```

Reply:
done.